exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -133,9 +136,12 @@ func (p *Poloniex) SetDefaults() {
 		},
 	}
 
-	p.Requester = request.New(p.Name,
+	p.Requester, err = request.New(p.Name,
 		common.NewHTTPClientWithTimeout(exchange.DefaultHTTPTimeout),
 		request.WithLimiter(SetRateLimit()))
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
```

Comment:
Could you please go into more detail about what's happened with the requester? You now cannot reuse a requester, that's fine, but when did that happen and if there is an error returned here, does that mean that an exchange doesn't have the ability to request? It seems like there's no recovery and returning the error would be better

Reply:
It will panic if the requester is not set that is correct. I have added a little bit more on the error return. I think if we are going to return an error on setdefaults() it might be better suited in a separate PR?

Reply:
Fair to not refactor setDefaults, but I still want to understand how this situation arises. If you request a requester on setDefaults and its in use already, why not create a new one and return that instead?  This could still be logged out so you have awareness of when this happens. Can it happen here now with the unloading?